aws/src/main/java/org/apache/iceberg/aws/dynamodb/DynamoDbCatalog.java

```diff
@@ -84,7 +84,12 @@
 import software.amazon.awssdk.services.dynamodb.model.TransactWriteItemsRequest;
 import software.amazon.awssdk.services.dynamodb.model.UpdateItemRequest;
 
-/** DynamoDB implementation of Iceberg catalog */
+/**
+ * DynamoDB implementation of Iceberg catalog
+ *
+ * @deprecated since version 1.6.0, will be removed after two full releases.
```

Comment:
should we say fully remove in 2.0 instead? That is a better version to introduce breaking changes.

Reply:
Yeah this would be a much cleaner approach. If we deprecate by 2.0 we increase the window for the plan forward.

Reply:
Yeah 2.0 is probably a better time to make this change.